### DISCUSSION TITLE
PnP Search Restriction on Restricted Content

### DISCUSSION THREAD

**Discussion by salil30july** [microsoft-search/pnp-modern-search#3508] [Q&A] [answered]
Hello Team,

I would to know if PnP Search Webpart restrict the content for current user. I mean if user won't have access to any site or content (restricted content), can he/she will be able to in result or not?

We know that SharePoint search service by default restrict the content based on user permission but I am not sure for this webpart as GRAPH APIs are having access to whole tenant.

I will look forward to your response and appreciate for quick response.

Regards,
Salil

**Comment by kasperbolarsen** [comment, marked as answer]
All of the requested (nor required) Graph Permissions are delegated permissions, PnP Modern Search does not hold any permissions at all, it is all based on the users permissions :-)

**Reply by salil30july** [reply]
Perfect! Thanks for your response!
